app/views/searches/show.html.erb

```diff
@@ -6,28 +6,44 @@
   </div>
 <% end %>
 <%= link_to t("advanced_search"), advanced_search_path, class: "t-link--gray t-link--has-arrow" %>
-<% if @gems %>
+<% if params[:yanked] %>
```

Comment:
this condition should exist in controller
https://stackoverflow.com/questions/15823695/best-practice-passing-instance-variables-or-using-params-in-ruby-on-rails-view